rke/templates/templates.go

```diff
@@ -100,17 +100,17 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.16.0-alpha": weavev18,
 		},
 		kdm.NginxIngress: {
-			">=1.8.0-rancher0 <1.13.10-rancher1-3":   nginxIngressv18,
-			">=1.13.10-rancher1-3 <1.14.0-rancher0":  nginxIngressV115,
-			">=1.14.0-rancher0 <=1.14.6-rancher1-1":  nginxIngressv18,
-			">=1.14.6-rancher2 <1.15.0-rancher0":     nginxIngressV115,
-			">=1.15.0-rancher0 <=1.15.3-rancher1-1":  nginxIngressv18,
-			">=1.15.3-rancher2 <1.15.12-rancher1-2":  nginxIngressV115,
-			"=1.15.12-rancher1-2":                    nginxIngressV11512,
-			">1.15.12-rancher1-2 <1.16.9-rancher1-2": nginxIngressV115,
-			"=1.16.9-rancher1-2":                     nginxIngressV11512,
-			">1.16.9-rancher1-2 <1.17.5-rancher1-2":  nginxIngressV115,
-			">=1.17.5-rancher1-2":                    nginxIngressV11512,
+			">=1.8.0-rancher0 <1.13.10-rancher1-3":    nginxIngressv18,
+			">=1.13.10-rancher1-3 <1.14.0-rancher0":   nginxIngressV115,
+			">=1.14.0-rancher0 <=1.14.6-rancher1-1":   nginxIngressv18,
+			">=1.14.6-rancher2 <1.15.0-rancher0":      nginxIngressV115,
+			">=1.15.0-rancher0 <=1.15.3-rancher1-1":   nginxIngressv18,
+			">=1.15.3-rancher2 <1.15.12-rancher1-2":   nginxIngressV115,
+			"=1.15.12-rancher1-2":                     nginxIngressV11512,
```

Comment:
`>= 1.15.12-rancher1-2` - - Any new template in k8s 1.15 should start using the updated nginx ingress template

Reply:
fixed